src/kernel/arch/aarch64/tty.zig

```diff
@@ -45,7 +46,7 @@ const font = [_][]const u1{
 var framebuffer: Framebuffer = undefined;
 
 fn writePixel(x: usize, y: usize, pixel: Pixel) void {
-    log.logDebug("Writing pixel {} to ({}, {}) at fb {}, which is address {x}\n", .{ pixel, x, y, @ptrToInt(framebuffer.buffer), @ptrToInt(&framebuffer.buffer[y * framebuffer.columns + x]) });
+    log(.debug, "Writing pixel {} to ({}, {}) at fb {}, which is address {x}\n", .{ pixel, x, y, @ptrToInt(framebuffer.buffer), @ptrToInt(&framebuffer.buffer[y * framebuffer.columns + x]) });
```

Comment:
Once you've started using the standard library logger this can be changed to `log.debug(...)`.

Reply:
Done